### PR TITLE
Added logic for users in zip codes with split congressional districts

### PR DESCRIPTION
**Pull request by cdoremus**
Logic added for users in zip codes with split congressional districts. 

Other things done in this PR:
* Used Trans from `i18n-next` to interpolate i18n message resources containing links
* Added clean-build-css script to `package.json` to do a clean CSS build from SCSS files
* Changed `LocationActionType` to string-based enum to allow code completion in editor
* A few additions/corrections to `README.md`

Please run `yarn` to install the new dependency `rimraf` and then `yarn clean-build-css`
to rebuild the CSS.

This PR is a partial implementation of issue #10 and a replacement for PR #32

### CODE REVIEW

**Comment by cdoremus** [comment]
The `tslint` script I added in commit 4511297 runs in the `test` script, so CircleCi will now fail when it runs `yarn test` and there are linting errors.

**Comment by nickoneill** [comment]
Testing this is bringing out some edge cases in location-finding:

1. Deny geolocation and get a backup from ipinfo, then reload the page. It looks like a timeout waits until browser geolocation fails again before fetching issues. We should fetch issues immediately if there's a location, then we can always refresh if we get a more accurate location later.

2. Regardless of browser location accept/deny, it looks like we're falling back and overriding manually-set location. For example, try entering a split zip (or any other zip) like 19444 - reload the page and the location with be reset to either browser location or ipinfo fallback. Manual location setting should never be overridden, in fact the codepath to try browser location or ipinfo fallback shouldn't even be attempted unless there's no existing location data.

File: src/redux/location/action.ts [hunk 1]
```diff
@@ -2,45 +2,50 @@ import { Action } from 'redux';
...
 
-export type LocationActionType =
-  'LOCATION_CLEAR' |
-  'LOCATION_SET' |
-  'CACHE_CITY' |
-  'NEW_LOCATION_LOOKUP' |
-  'SET_UI_STATE' |
-  'SET_LOCATION_FETCH_TYPE'
-;
+export enum LocationActionType {
```
Comment (nickoneill) [inline src/redux/location/action.ts:5]: This is excellent, I like getting away from the stringiness here

File: src/components/done/__snapshots__/Done.test.tsx.snap [hunk 2]
```diff
@@ -35,11 +35,19 @@ ShallowWrapper {
         />
         <p
             className="call__text"
-        />
-        <strong>
-            TOWN HALL COMPONENT HERE
-        </strong>
-        <br />
+        >
+            <Trans
```
Comment (nickoneill) [inline src/components/done/__snapshots__/Done.test.tsx.snap:39]: What prompted this change and including the `<0>` and `<1>` parts in the translation files? Feels like these things are getting tightly coupled with the extra required parameters

File: README.md [hunk 3]
```diff
@@ -16,35 +16,32 @@ To build the application, you need to install [Yarn](https://yarnpkg.com/) and r
...
+yarn start:https
 
 # build the app into build folder
 #  for server deployment:
 yarn build
 
 # any updates to .scss files need
 #  to be compiled to css using:
-yarn build-css
+yarn clean-build-css
```
Comment (nickoneill) [inline README.md:39]: Can we run the clean css command before every build as well to get rid of those duplicate css errors when just running straight up `yarn`?

Comment (cdoremus) [reply README.md:39]: Fixed in commit c13d12f

File: src/components/done/__snapshots__/Done.test.tsx.snap (hunk 2 above)
Comment (cdoremus) [reply src/components/done/__snapshots__/Done.test.tsx.snap:39]: This syntax is necessary to do string interpolation with the `react-i18next` library.

**Comment by cdoremus** [comment]
@nickoneill the fixes to the issues you itemimzed above are in commit 5bdf517 (1) and commit 5470dec (2)

**Comment by nickoneill** [comment]
Looks great! Merge when ready

**Review by nickoneill — APPROVED** [review]
